src/transformed_distribution.jl

```diff
@@ -33,8 +33,10 @@ transformed(d) = transformed(d, bijector(d))
 
 Returns the constrained-to-unconstrained bijector for distribution `d`.
 """
-bijector(d::Distribution) = DistributionBijector(d)
-bijector(d::UnivariateDistribution) = TruncatedBijector(minimum(d), maximum(d))
+bijector(d::DiscreteUnivariateDistribution) = Identity{0}()
+bijector(d::ContinuousUnivariateDistribution) = TruncatedBijector(minimum(d), maximum(d))
+bijector(d::Distributions.Product) = stack(bijector.(d.v)...)
```

Comment:
This splatting is not good for large arrays commonly used with `Product` in Turing.

Reply:
I'm fine with not splatting, i.e. doing `Stacked(bijector.(d.v))`, but to make things type-stable it seems like we have to splat, right? Or equivalently do `Stacked(tuple(bijector.(d.v)))` but that should be the same?

Reply:
I don't think type stability is a concern here since all of the elements of `d.v` are usually of the same type. `tuple` should be equally bad ya.

Reply:
Actually, `tuple` doesn't convert, `Tuple` does.

Reply:
> I don't think type stability is a concern here since all of the elements of d.v are usually of the same type.

Ah, if that's the case then it should be fine :+1: